sdk/core/azure-core/test/ut/proxy_tests/remoteproxy.passwd/Dockerfile

```diff
@@ -0,0 +1,30 @@
+#@# vim: set filetype=dockerfile:
+# Code taken from: https://github.com/hinata/nginx-forward-proxy
+
+FROM ubuntu/squid
+LABEL maintainer "Larry Osterman<github.com/LarryOsterman>"
```

Comment:
Just to confirm, the label in a Dockerfile is just metadata, correct, and doesn't have much semantic meaning? If so, any name/string like this seems good to me. Otherwise, we may want to use some other Azure SDK GitHub account :)

Reply:
It is just metadata. If this dockerfile was to be put in a public docker repository and someone downloaded it, this would let them know who to get in touch with. But this dockerfile should never go into a public repository.